lib/cutest.rb

```diff
@@ -148,19 +148,19 @@ def test(name = nil, &block)
     cutest[:test] = nil
   end
 
-  # Assert that value is not nil or false.
-  def assert(value)
-    flunk("expression returned #{value.inspect}") unless value
+  # Assert that test is a true value.
+  def assert(test)
+    flunk("expression returned #{test.inspect}") unless test
     success
   end
 
-  # Assert that two values are equal.
-  def assert_equal(value, other)
-    flunk("#{value.inspect} != #{other.inspect}") unless value == other
+  # Assert that expected and actual values are equal.
+  def assert_equal(expected, actual)
```

Comment:
Sometimes it actually matters which side has the expected value and which side has the actual value. This is because `assert_equal` uses `#==` for equality testing; `a == b` returning `true` doesn't necessarily mean `b == a` will also return `true`. This is Ruby after all.

I think it's best to keep the argument names as they are. They're closest to how `#==` in core names its argument.